models/reddit_ads.yml

```diff
@@ -122,14 +122,14 @@ models:
             - campaign_id
             - currency
     columns:
-      - name: date_day
-        description: '{{ doc("date_day") }}'
+      - name: date_day 
+        description: '{{ doc("date_day") }}' 
```

Comment:
Should we apply not_null tests to these fields?

Reply:
updated